src/cmd/manage_nsfs.js

```diff
@@ -0,0 +1,452 @@
+/* Copyright (C) 2020 NooBaa */
+'use strict';
+
+const dbg = require('../util/debug_module')(__filename);
+const nb_native = require('../util/nb_native');
+const fs_utils = require('../util/fs_utils');
+const SensitiveString = require('../util/sensitive_string');
+const minimist = require('minimist');
+const path = require('path');
+const fs = require('fs');
+const P = require('../util/promise');
+const _ = require('lodash');
+const config = require('../../config');
+
+const HELP = `
+Help:
+
+    "nsfs" is a noobaa-core command runs a local S3 endpoint on top of a filesystem.
+    Each sub directory of the root filesystem represents an S3 bucket.
+    manage nsfs will provide a command line interface to create new accounts and map existing directories 
+    to Noobaa as buckets. For more information refer to the noobaa docs.
+`;
+
+const USAGE = `
+Usage:
+
+    node src/cmd/manage_nsfs <action> <type> [options...]
+`;
+
+const ARGUMENTS = `
+Arguments:
+
+    <type>    Set the resource type such as accounts and buckets
+    <action>  Action could be add, update, list, status and delete for accounts/buckets.
+`;
+
+const ACCOUNT_OPTIONS = `
+Account Options:
+
+    # Read account details from json file, no need to mention all the propery one by one in CLI
+    --from_file <dir>           (default none)          Set buckt/account schema full path.
+                                                        Get bucket and account details from json file                
+
+    # required for add, replace 
+    --name <name>               (default none)          Set the name for account/bucket.
+    --email <email>             (default none)          Set the email for account/bucket.
+    --uid <uid>                 (default as process)    Send requests to the Filesystem with uid.
+    --gid <gid>                 (default as process)    Send requests to the Filesystem with gid.
+    --secret_key <key>          (default none)          The secret key pair for the access key.
+    --new_buckets_path <dir>    (default none)          Set the root of the filesystem where each subdir is a bucket.
+    
+    # required for add, replace and delete
+    --access_key <key>          (default none)          Authenticate incoming requests for this access key only (default is no auth).
+    --config_root <dir>         (default none)          Configuration files path for Noobaa standalon NSFS.
+`;
+
+const BUCKET_OPTIONS = `
+Bucket Options:
+
```

Comment:
I think it's important here to remind the user that create bucket cli command won't create the directory and the bucket directory should exist prior to the create bucket cli command

Reply:
@romayalon but we are creating the bucket for the user from cli in method `check_and_create_config_dirs()`, Should I remove this?

Reply:
@romayalon oh.. ok, I got confused it with config_root dir, Yes I will mention about bucke dir in doc, Thank you